onnx/defs/printer.cc

```diff
@@ -216,6 +215,8 @@ void ProtoPrinter::print(const TensorProto& tensor, bool is_initializer) {
         output_ << "..."; // ParseData not instantiated for other types.
         break;
     }
+  } else if (tensor.has_data_location() && tensor.data_location() == onnx::TensorProto_DataLocation_EXTERNAL) {
```

Comment:
```suggestion
  } else if (tensor.has_data_location() && tensor.data_location() == TensorProto_DataLocation_EXTERNAL) {
```

We cannot reference the onnx:: namespace directly because it is compile time defined.

Reply:
Thank you. I don't know that. It remands me that I should read the contributing document. Now I had read the `CONTRIBUTING.md` and found I also need to run the lint.